docs/src/further_topics/ugrid/operations.rst

```diff
@@ -0,0 +1,953 @@
+        ...         "NetCDF",
+        ...         "unstructured_grid",
+        ...         "lfric_ngvat_2D_72t_face_half_levels_main_conv_rain.nc",
+        ...     ]
+        ... )
+
+        >>> with PARSE_UGRID_ON_LOAD.context():
+        ...     global_cube = load_cube(file_path, "conv_rain")
+        >>> print(global_cube)
+        surface_convective_rainfall_rate / (kg m-2 s-1) (-- : 72; -- : 864)
+            Mesh coordinates:
+                latitude                                    -        x
+                longitude                                   -        x
+            Auxiliary coordinates:
+                time                                        x        -
+            Cell methods:
+                point                                   time
+            Attributes:
+                Conventions                             UGRID
+                description                             Created by xios
+                interval_operation                      300 s
+                interval_write                          300 s
+                name                                    lfric_ngvat_2D_72t_face_half_levels_main_conv_rain
+                online_operation                        instant
+                timeStamp                               2020-Oct-18 21:18:35 GMT
+                title                                   Created by xios
+                uuid                                    b3dc0fb4-9828-4663-a5ac-2a5763280159
+
+        # Convert the Mesh to a GeoVista PolyData object.
+        >>> lons, lats = global_cube.mesh.node_coords
+        >>> face_node = global_cube.mesh.face_node_connectivity
+        >>> indices = face_node.indices_by_location()
+        >>> global_polydata = Transform.from_unstructured(
+        ...     lons.points, lats.points, indices, start_index=face_node.start_index
+        ... )
+
+        # Define a region of 4 corners connected by great circles.
+        #  Specialised sub-classes of BBox are also available e.g. panel/wedge.
+        >>> region = BBox(lons=[0, 70, 70, 0], lats=[-25, -25, 45, 45])
+        # 'Apply' the region to the PolyData object.
+        >>> region_polydata = region.enclosed(global_polydata, preference="center")
+        # Get the remaining face indices, to use for indexing the Cube.
+        >>> indices = region_polydata["vtkOriginalCellIds"]
+
+        >>> print(type(indices))
+        <class 'numpy.ndarray'>
+        # 101 is smaller than the original 864.
+        >>> print(len(indices))
+        101
+        >>> print(indices[:10])
+        [ 6  7  8  9 10 11 18 19 20 21]
+
+        # Use the face indices to subset the global cube.
+        >>> region_cube = global_cube[:, indices]
+
+        # In this case we **know** the indices correspond to a contiguous
+        #  region, so we will convert the sub-setted Cube back into a
+        #  Cube-with-Mesh.
+        >>> new_mesh = Mesh.from_coords(*region_cube.coords(dimensions=1))
```

Comment:
This seems almost a "sly" introduction of from_coords.
Which is nevertheless a potentially really useful operation to the user.
Is there anyway of highlighting this a bit more, I wondered ?

Reply:
970db3ca5ea0b1c32a3ae7730b51a1df7db0d847